### PR TITLE
Support Spark Reader 2.0

### PR DESCRIPTION
**Pull request by Nicole00**
Need to synchronize the data structure in the scan result when pr #18 is merged.  

data structure has been synchronized.

### CODE REVIEW

File: nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/NebulaOptions.scala [hunk 1]
```diff
@@ -142,7 +142,11 @@ class NebulaOptions(@transient val parameters: CaseInsensitiveMap[String])(
   }
 
   def getReturnCols: List[String] = {
-    returnCols.split(",").toList
+    if (returnCols.trim.isEmpty) {
```
Comment (darionyaphet) [inline nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/NebulaOptions.scala:145]: if `returnCols` is empty why don't report an error？

File: nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/package.scala [hunk 2]
```diff
@@ -144,45 +147,70 @@ package object connector {
...
-              props.append(NebulaUtils.resolveDataAndType(row, cols(i).dataType, i))
+              props.append(row.get(i))
             }
           }
+          val srcId = row.get(0).toString.getBytes()
+          val dstId = row.get(1).toString.getBytes()
+          val edgeSrc = if (row.schema.fields(0).dataType == LongType) {
+            srcId.toString.toLong
+          } else {
+            MurmurHash2.hash64(srcId, srcId.length, 0xc70f6907)
```
Comment (darionyaphet) [inline nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/package.scala:203]: `0xc70f6907 ` is a magic number use to make sure the hash value same with nebula engine ？

File: nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/NebulaOptions.scala (hunk 1 above)
Comment (Nicole00) [reply nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/NebulaOptions.scala:145]: Empty returnCols means return all properties, keep consistent with client and storage interface.

File: nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/package.scala (hunk 2 above)
Comment (Nicole00) [reply nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/package.scala:203]: Yeah, only when use this seed `0xc70f6907 ` can keep the same value with Nebula.

**Review by jude-zhu — APPROVED** [review]

**Pull request merged**